src/app/deeds/deeds.component.ts

```diff
@@ -46,8 +53,11 @@ export class DeedsComponent implements OnInit {
   }
 
   openTeamRegistration(deed: Deed) {
-    if (this.isLoggedIn) {
-      const dialogRef = this.dialog.openTeamDialog(deed);
+    if (this.userService.loggedInStatus) {
```

Comment:
if (this.userService.isLoggedIn) would be better. If method returns bolean you should start naming it with is